powerauth-admin/src/main/java/io/getlime/security/app/admin/controller/ActivationController.java

```diff
@@ -66,7 +63,7 @@ public ActivationController(PowerAuthClient client) {
      * @param model Model with passed parameters.
      * @return "activations" view.
      */
-    @RequestMapping(value = "/activation/list")
+    @RequestMapping("/activation/list")
```

Comment:
I am not sure these `@RequestMapping`s are correct in the original code. We should probably specify a HTTP method here. We don't want this endpoint to be accessible via `DELETE` method. This issue is present in other classes, too.

Reply:
👍 But let's do it in a separate PR. It will be easier to review the code and better for history tracking.